README.md

```diff
@@ -108,3 +106,5 @@ with AlertConsumer(topics, config) as consumer:
     topic, alert = consumer.poll(5)
     print("topic: {}, alert:\n {}".format(topic, alert))
 ```
+
+To learn more about the schema of received alerts and how to work with them, refer to the [jupyter notebook](notebooks/working_with_alerts.ipynb).
```

Comment:
Could you add WIP at the end of the sentence? I will update it in a future PR.

Reply:
done in 7be307a.